public/app/screens/main/devices/device-list/device-list.jsx

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+var React = require("react");
+var Device = require('./device');
+var Room = require('./room');
+
+var DeviceList = React.createClass({
```

Comment:
Actually it's just a placeholder since we don't have logic there, here I wanted to test interaction between components. Probably you are right, we might not need it in future.